src/main/java/net/pms/dlna/DLNAMediaInfo.java

```diff
@@ -877,6 +877,7 @@ public void parse(InputFile inputFile, Format ext, int type, boolean thumbOnly,
 					// Create the thumbnail image using the Thumbnailator library
 					try {
 						thumb = ImagesUtil.scaleImage(Files.readAllBytes(file.toPath()), 320, 320, ScaleType.MAX, ImageFormat.JPEG, false);
+						ffmpeg_parsing = false;
```

Comment:
@valib Yes, I've seen that but I haven't quite understood the logic. To me it seems like the bug is there, it shouldn't matter if we should use MPlayer for video thumbnails when we're processing an image..?